### DISCUSSION TITLE
i can't get Status Code

### DISCUSSION THREAD

**Discussion by keene-dalveytech** [axios/axios#5864] [Q&A]
### Describe the bug

<img width="654" alt="image" src="https://user-images.githubusercontent.com/126315969/233533238-0bc00e0a-8174-4039-a56c-1c4c342e82d6.png">
<img width="460" alt="image" src="https://user-images.githubusercontent.com/126315969/233533290-63d39db7-fb9c-43c9-b834-d0cdfcfa4120.png">

**Comment by Mihir99-mk** [comment]
#5668 This status code occurs when a user requested a resource that is not available due to legal reasons
 (Unavailable For Legal Reasons).

**Comment by lleitep3** [comment]
Same problem here

**Comment by LokeshKumar-Dev** [comment]
Bro... It gets resolved

**Comment by lleitep3** [comment]
just to update, in my case it was CORS problem, not exactly axios

**Comment by jasonsaayman** [comment]
Closing as solved
